.github/actions/build/action.yaml

```diff
@@ -47,11 +47,11 @@ runs:
       # TODO - needs to be smarter!
       run: |
         if [ "${{ inputs.build-type }}" = "release" ]; then
-          echo "LMS_TAG=--tag=rc" >> $GITHUB_OUTPUT
+          echo "LMS_TAG=--tag=rc,${{ steps.getversion.outputs.LMS_FULL_VERSION }}-rc" >> $GITHUB_OUTPUT
```

Comment:
Please do not modify this tag, as it's not for public consumption. It should be replaced with the actual release (`latest`) tag within a few minutes. I only put it there to make sure users wouldn't pick up a new `latest` before all builds have been done and the release process has succeeded.